docs/example-data.json

```diff
@@ -53,18 +53,16 @@
         "supported_motion_$_ids": [],
```

Comment:
Just a thought and does not belong to this PR, but should we build a validator for the example data? It would prevent errors such as https://github.com/OpenSlides/openslides-backend/issues/324 and shouldn't take to long on the basis of the `models.yml`. It would be especcially easy with the existing backend code (I think) but it should belong to this repo, like the models validator. What do you think, how long would this take in go, maybe an the basis of the models validator? I think it would be a helpful addition.